PivotSubscriptions/Task.cls

```diff
@@ -11,22 +11,26 @@ Method OnTask() As %Status
 	Set tDayOfWeek=$zd($h,10)
 	Set tHour=$p($zt($piece($h,",",2),2),":",1)
 	Set tMinute=$p($zt($piece($h,",",2),2),":",2)
-	Set tRS=##class(%SQL.Statement).%ExecDirect(,"SELECT Emails,Pivot FROM PivotSubscriptions.Subscription WHERE DayOfWeek[? AND ""Hour""=? AND ""Minute""=?",tDayOfWeek,tHour,tMinute)
+	Set tRS=##class(%SQL.Statement).%ExecDirect(,"SELECT Emails,Pivot,Format FROM PivotSubscriptions.Subscription WHERE DayOfWeek[? AND ""Hour""=? AND ""Minute""=?",tDayOfWeek,tHour,tMinute)
 	
 	// Loop through Subscription results and send emails.
 	While tRS.%Next() {
 		Set tEmails=tRS.%Get("Emails")
 		Set tPivot=tRS.%Get("Pivot")
+		Set tFormat=tRS.%Get("Format")
 		
-		Do ..SendEmail(tPivot,tEmails)
+		Do ..SendEmail(tPivot,tEmails,tFormat)
 	}
 	
 	Quit tSC
 }
 
-ClassMethod SendEmail(pPivot As %String, pEmails As %String) As %Status
+// unit test manager

```

Comment:
Did this comment get included accidentally?

Reply:
I believe that comment was in the original code. I deleted it at some point, but put it back when I was separating the email methods from the Export PDF feature.